schemaless/sources.py

```diff
@@ -43,7 +43,7 @@ def get_value(self, record):
         try:
             return datetime.strptime(
                 record[self.field].split(" ")[0], self.date_format).date()
-        except:
+        except Exception:
```

Comment:
Catching a base `Exception` is generally recommended against. Are you intending to capture a parse error, or a string error or something?

One reason this is recommended against is because this will also capture a keyboard interrupt, so rather than exit the program it continues.

Reply:
Yeah it's an error I hit when attempting to parse the date. There was a case with some PPTS data I loaded in where it was trying to load "15CAP-00000-00AUJ" as a date and threw a ValueError.

I realize after your message this morning though that I probably hit that because of the malformed data that got loaded into testdata. Should I just except a ValueError to be safe then?

Reply:
Yeah, looks like it throws a ValueError, so catch that.